2021-12-13 19:50:24,285 INFO    MainThread:11092 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 19:50:24,285 INFO    MainThread:11092 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 19:50:24,285 INFO    MainThread:11092 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_195024-pzipc7ya\logs\debug.log
2021-12-13 19:50:24,285 INFO    MainThread:11092 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_195024-pzipc7ya\logs\debug-internal.log
2021-12-13 19:50:24,286 INFO    MainThread:11092 [wandb_init.py:init():404] calling init triggers
2021-12-13 19:50:24,286 INFO    MainThread:11092 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 19:50:24,288 INFO    MainThread:11092 [wandb_init.py:init():449] starting backend
2021-12-13 19:50:24,288 INFO    MainThread:11092 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 19:50:24,293 INFO    MainThread:11092 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 19:50:24,409 INFO    MainThread:11092 [backend.py:ensure_launched():204] started backend process with pid: 9580
2021-12-13 19:50:24,409 INFO    MainThread:11092 [wandb_init.py:init():458] backend started and connected
2021-12-13 19:50:24,411 INFO    MainThread:11092 [wandb_init.py:init():519] updated telemetry
2021-12-13 19:50:24,444 INFO    MainThread:11092 [wandb_init.py:init():549] communicating current version
2021-12-13 19:50:25,062 INFO    MainThread:11092 [wandb_init.py:init():554] got version response 
2021-12-13 19:50:25,062 INFO    MainThread:11092 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 19:50:25,303 INFO    MainThread:11092 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 19:50:25,451 INFO    MainThread:11092 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 19:50:25,452 INFO    MainThread:11092 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 19:50:25,452 INFO    MainThread:11092 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 19:50:25,454 INFO    MainThread:11092 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 19:50:25,454 INFO    MainThread:11092 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 19:50:25,455 INFO    MainThread:11092 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000029952F34B20>>
2021-12-13 19:50:25,455 INFO    MainThread:11092 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2021-12-13 19:50:25,456 INFO    MainThread:11092 [wandb_config.py:__setitem__():144] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000029952F34B20>>
2021-12-13 19:50:25,456 INFO    MainThread:11092 [wandb_run.py:_config_callback():962] config_cb batch_size 32 None
2021-12-13 19:50:26,930 INFO    MainThread:11092 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 19:50:26,930 INFO    MainThread:11092 [wandb_run.py:_restore():1758] restore
2021-12-13 19:50:27,482 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:50:27,587 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:50:27,806 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:50:28,341 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3631
}

2021-12-13 19:50:28,450 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3657
}

2021-12-13 19:50:28,560 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3657
}

2021-12-13 19:50:28,669 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:28,776 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:28,887 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:28,996 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:29,106 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:29,216 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:29,326 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:29,727 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}

2021-12-13 19:50:30,258 INFO    MainThread:11092 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3657
  total_bytes: 3657
}
local_info {
}

2021-12-13 19:50:31,345 INFO    MainThread:11092 [wandb_run.py:_append_files():2184] logging synced files
